### ISSUE TITLE
Battleye in 2021.

### ISSUE THREAD

**Issue by EvelynARK** [arkmanager/ark-server-tools#1206]
Tried adding the flag "arkflag_UseBattlEye" and it returned no successful results in removing battleye.
Has this been changed over time? If so, what is the new flag please? We have a player that can't connect because battleye quite literally refuses to work for them even after numerous reinstalls and whatnot.

Many thanks.

**Comment by klightspeed** [comment]
If I recall correctly, `-UseBattlEye` was replaced with `-NoBattlEye` sometime in 2016 when Wildcard turned BattlEye on by default.

**Comment by Nightreaver** [comment]
Yes, i just disabled it for my server with `arkflag_NoBattleye=true` in my config file
